lib/matplotlib/_internal_utils.py

```diff
@@ -0,0 +1,52 @@
+"""
+Internal debugging utilities, that are not expected to be used in the rest of
+the codebase.
+
+WARNING: Code in this module may change without prior notice!
+"""
+
+from io import StringIO
+from pathlib import Path
+import subprocess
+
+from matplotlib.transforms import TransformNode
+
+
+def graphviz_dump_transform(transform, dest, *, highlight=None):
+    """
+    Generate a graphical representation of the transform tree for *transform*
+    using the :program:`dot` program (which this function depends on).  The
+    output format (png, dot, etc.) is determined from the suffix of *dest*.
+    """
```

Comment:
Even though this is not user-facing, a parameter description would be helpful, e.g.

~~~
Parameters
----------
transform : `~matplotlib.transform.Transform`

dst : str
    filename of output file. This must be one of the formats supported
    by :program:`dot`, e.g. png, svg, ... (see https://www.graphviz.org/doc/info/output.html).
highlight : list of `~matplotlib.transform.Transform`
    The transforms in the tree to be highlighted (drawn in bold).
    If *None*, *transform* is highlighted.
~~~

Reply:
done